src/main/java/mate/academy/service/CinemaHallService.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.service;
+
+import java.util.List;
+import mate.academy.model.CinemaHall;
+
+public interface CinemaHallService {
+    CinemaHall add(CinemaHall cinemaHall);
+
+    CinemaHall get(Long id);
```

Comment:
According to the checklist, the `get()` method in the service layer should return the entity itself, not an `Optional`. However, the `get()` method on the dao layer should return `Optional`. Please, make sure that this is correctly implemented in the corresponding dao implementation.